codenames-domain/src/main/java/com/codenames/mapper/PlayerMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.codenames.mapper;
+
+import com.codenames.dto.UserDto;
+import com.codenames.models.game.Player;
+import org.mapstruct.IterableMapping;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+import java.util.List;
+
+@Mapper
+public interface PlayerMapper {
+
+    TeamMapper INSTANCE = Mappers.getMapper(TeamMapper.class);
```

Comment:
There is a `uses` param in `@Mapper` annotation that can be used to inject other mappers